components/layout/navbar/profile-menu/profile-item.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import LogOut from "@/public/log-out.svg";
+import Setting from "@/public/setting.svg";
+import Arrow from "@/public/arrow-menu.svg";
+import { signOut, useSession } from "next-auth/react";
+import { MenuContent } from "@/types/ui-content";
+import Image from "next/image";
+
+type ProfileItemProps = {
+  menu: MenuContent[];
+};
+
+export const ProfileItem = ({ menu }: ProfileItemProps) => {
+  const { data: session } = useSession();
+  const admin = session?.user?.role === "admin";
+  const year = new Date().getFullYear();
+
+  const adminMenu = menu.find(
+    (item) => item.title === ("Administration" || "Admin"),
+  );
+  const signOutMenu = menu.find(
+    (item) => item.title === ("Déconnexion" || "Sign Out"),
```

Comment:
I don't understand what's the point of this, why not simply use a map to show the complete menu? This makes adding new element to menu complicated and we'll need to change the code for every client that wan't a new tab, modify this component.

Reply:
i explained to you why the other day, we will take about it tomorrow